packages/api-plugin-fulfillment/src/mutations/createFulfillmentMethod.js

```diff
@@ -0,0 +1,63 @@
+import SimpleSchema from "simpl-schema";
+import Random from "@reactioncommerce/random";
+import ReactionError from "@reactioncommerce/reaction-error";
+import { FulfillmentMethodSchema } from "../simpleSchemas.js";
+
+const inputSchema = new SimpleSchema({
+  method: FulfillmentMethodSchema,
+  fulfillmentTypeId: String,
+  shopId: String
+});
+
+/**
+ * @method createFulfillmentMethodMutation
+ * @summary Creates a fulfillment method
+ * @param {Object} context - an object containing the per-request state
+ * @param {Object} input - Input
+ * @param {Number} cost - optional cost
+ * @param {String[]} fulfillmentTypes - fulfillment type retained for backward compatibility
+ * @param {String} group - Group to which fulfillment method belong
+ * @param {Number} handling - handling charges
+ * @param {Boolean} enabled - status of fulfillment method
+ * @param {String} label - displayed on the UI
+ * @param {String} fulfillmentMethod - used by application, not user editable
+ * @param {String} displayMessageMethod - used to display additional message on UI specific to ff-method
+ * @param {Number} rate - ratefor themethod
```

Comment:
These field names should clarify the full path to the property. Example, `input.fulfillmentTypeId` or `input.method.cost`

Reply:
Added